modules/bash-commons/src/aws.sh

```diff
@@ -20,36 +20,65 @@ readonly three_hours_in_s=10800
 # a token from IMDSv2
 readonly six_hours_in_s=21600
 
-# Detect if the instance is using IMDSv2 or if it is using IMDSv1 still.
+# shellcheck source=./modules/bash-commons/src/assert.sh
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/assert.sh"
+# shellcheck source=./modules/bash-commons/src/log.sh
```

Comment:
I didn't see any reason why these source statements couldn't be higher in the script.